### DISCUSSION TITLE
How to customise the grid with the plotting definitions?

### DISCUSSION THREAD

**Discussion by LaughingMaann** [colour-science/colour#1263] [Q&A]
### Question

want to plot major grid lines similar to the left (example matplotlib), onto the plot_planckian_locus_in_chromaticity_diagram_CIE1931 (desired to the right). specifically, i wanted the grid lines dashed, to show for every chromaticity step size of 0.1, both in the x-axis and y-axis.

boxed in the right, i've tried to do just about everything but cannot get the grid lines to show like the left screeenshot example plot. please halp me senpai
![image](https://github.com/colour-science/colour/assets/16868265/10f39966-a5a5-4af0-b74b-aa425e192265)

**Comment by thomasmansencal** [comment]
Hello @LaughingMaann,

Try something like that:

```python
import colour

figure, axes = colour.plotting.plot_chromaticity_diagram_CIE1931(show=False)

axes.grid(color="red", linestyle="--", linewidth=0.5)

colour.plotting.render()
```

Cheers,

Thomas
